### DISCUSSION TITLE
Search for an article linking to the wrong category

### DISCUSSION THREAD

**Discussion by Lolabird** [joomla/joomla-cms#38189] [Q&A]
This seems to be an isolated incident, but I am having an issue with a search on a site I have been working has linked to the wrong category instead of the one that was set. It is a HWD video if that makes a difference. Would anyone happen to know why a search result would link to the wrong category and how that might be fixed?

**Comment by Hackwar** [comment]
For support questions like this, please go to forum.joomla.org
